NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -519,4 +520,82 @@ class NeuroID private constructor(
     }
 
     fun getSDKVersion() = NIDVersion.getSDKVersion()
+
+    // new Session Commands
+    fun clearSessionVariables() {
+        userID = ""
+//        registeredUserID = ""
+    }
+
+    fun startSession(sessionID: String = ""): SessionStartResult {
+        if (clientKey == "") {
+            NIDLog.e(
+                msg = "Missing Client Key - please call configure prior to calling start"
+            )
+            return SessionStartResult(false, "")
+        }
+
+        if (userID != "" || isSDKStarted) {
+            stopSession()
+        }
+
+        val finalSessionID = if (sessionID != null && sessionID != "") {
+            sessionID
+        } else {
+            generateUniqueHexId()
+        }
+
+        if (!setUserID(finalSessionID)) {
+            return SessionStartResult(false, "")
+        }
+
+        isSDKStarted = true
```

Comment:
Is this redundant since it is also set to true in the resumeCollection call?

Reply:
it is semi redundant but we needed the isSDKStarted variable to be true before createSession so it captures the events